src/eduid_webapp/support/views.py

```diff
@@ -30,39 +32,29 @@ def index(support_user):
                 return render_template('index.html', support_user=support_user,
                                        error="No users matched the search query")
 
-        current_app.logger.info('Support personnel: {!r} searched for {!r}'.format(support_user, search_query))
+        current_app.logger.info('Support personnel {} searched for {}'.format(support_user, search_query))
```

Comment:
Audit log _who_ searched too?

Reply:
support_user is the user that is logged in to the support app. Could use a better name perhaps.